lib/components/menu/menu-links.tsx

```diff
@@ -36,9 +38,17 @@ const MenuLinks = () => {
           height: 60px;
         }
 
-        .site-name {
+        .menu__logo {
           display: flex;
+          flex-direction: row;
           align-items: center;
+          font-size: 1.125rem;
+          font-weight: 500;
+          color: inherit;
+        }
+
+        .menu__logo :global(.avatar) {
+          margin-right: ${theme.layout.gapHalf}!important;
```

Comment:
`<Avatar mr="10px" />`

Reply:
@unix It'll not work since this:
 https://github.com/geist-org/react/blob/74e76adbcf97f0ce76bbb3d3e382be52fdf3cfdb/components/avatar/avatar.tsx#L83-L85

Reply:
Can you create a PR and fix this as well, now `Avatar` no longer needs to set children margin manually.